index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html/css_quiz</title>
+    <link rel="stylesheet" type="text/css" href="quiz.css">
+  </head>
+  <body>
+    <article>
+      <header id="pencil">
+        <h1>Pencil</h1>
+      </header>
+      <div>
+        <img src="https://pixabay.com/static/uploads/photo/2016/03/31/23/37/draw-1297723__180.png">
+        

```

Comment:
Careful of trailing whitespace. Whitespace will inevitably lead to merge conflicts. You can fix this by adding the Trailing Spaces package to your text editor or adding this line `"trim_trailing_white_space_on_save": true` to the config in your user settings.